packages/swapper/src/swappers/thorchain/utils/constants.ts

```diff
@@ -1,10 +1,31 @@
+import { KnownChainIds } from '@shapeshiftoss/types'
+import BigNumber from 'bignumber.js'
+
+import { bn } from '../../utils/bignumber'
+
 export const MAX_THORCHAIN_TRADE = '100000000000000000000000000'
 export const MAX_ALLOWANCE = '100000000000000000000000000'
 export const THOR_MINIMUM_PADDING = 1.2
 export const THOR_ETH_GAS_LIMIT = '100000' // for sends of eth / erc20 into thorchain router
 export const THORCHAIN_FIXED_PRECISION = 8 // limit values are precision 8 regardless of the chain
 
-// These are different from THOR_ETH_GAS_LIMIT
 // Used to estimate the fee thorchain will take out of the buyAsset
-export const THOR_TRADE_FEE_BTC_SIZE = 1000 // used for estimating thorchain side fees
-export const THOR_TRADE_FEE_ETH_GAS = 120000 // used for estimating thorchain side fees
+// Official docs on fees are incorrect
+// https://discord.com/channels/838986635756044328/997675038675316776/998552541170253834
+// This is still not "perfect" and tends to overestimate by a small randomish amount
+// TODO figure out if its possible to accurately estimate the outbound fee.
+// Neither the discord nor official docs are correct
+export const THOR_TRADE_FEE_MULTIPLIERS: Record<SUPPORTED_BUY_CHAINS, BigNumber> = {
+  [KnownChainIds.BitcoinMainnet]: bn(0.00002),
+  [KnownChainIds.DogecoinMainnet]: bn(0.00002),
+  [KnownChainIds.LitecoinMainnet]: bn(0.000005),
+  [KnownChainIds.CosmosMainnet]: bn(0.00000002),
+  [KnownChainIds.EthereumMainnet]: bn(0.00024)
```

Comment:
Non-blocking, but `toString()` these? Passing primitives around has a lot less overhead and serializes better. Although in this case it doesn't matter as we just `THOR_TRADE_FEE_MULTIPLIERS[buyChainId].times()` without any passing around, so your call

Reply:
I agree string an be useful in some cases but I don't think theres much benefit and would still need to be converted back to a bignumber for math and then back into a string